src/Conda.jl

```diff
@@ -369,4 +371,46 @@ function import_list(io::IO, args...; kwargs...)
     end
 end
 
+"""
+    pip_interop(bool::Bool, env::Environment=$ROOTENV)
+
+Sets the `pip_interop_enabled` value to bool.
+If `true` then the conda solver is allowed to interact with non-conda-installed python packages.
+"""
+function pip_interop(bool::Bool, env::Environment=ROOTENV)
+    runconda(`config --set pip_interop_enabled $bool --file $(conda_rc(env))`, env)
+end
+
+"""
+    pip_interop(env::Environment=$ROOTENV)
+
+Gets the `pip_interop_enabled` value from the conda config.
+"""
+function pip_interop(env::Environment=ROOTENV)
+    dict = parseconda(`config --get pip_interop_enabled --file $(conda_rc(env))`, env)["get"]
+    isempty(dict) ? false : dict["pip_interop_enabled"]
```

Comment:
`get(dict, "pip_interop_enabled", false)`, probably?

Reply:
I changed
```julia
isempty(dict) ? false : dict["pip_interop_enabled"]
```
to
```julia
get(dict, "pip_interop_enabled", false)
```